packages/react-router-dom/modules/Link.js

```diff
@@ -5,6 +5,9 @@ import invariant from 'invariant'
 const isModifiedEvent = (event) =>
   !!(event.metaKey || event.altKey || event.ctrlKey || event.shiftKey)
 
+const resolveToLocation = (to, location) =>
```

Comment:
I'm fine with any name) It was just a copy/paste from RR3 implementation. Could you suggest something? 'getToLocation'?